app/views/shared/_problem_order_details.html.haml

```diff
@@ -28,9 +28,7 @@
         %td.centered= order_detail.quantity
         %td= order_detail_description(order_detail)
         %td= order_detail.account
-        - if order_detail.requires_but_missing_actuals?
-          %td.centered= "#{t ".actual_usage_missing"} #{link_to "Enter", manage_order_detail_path(order_detail), class: "manage-order-detail"}".html_safe
-        - else
-          %td.centered= "#{t ".price_policy_missing"} #{link_to "Assign", manage_order_detail_path(order_detail), class: "manage-order-detail"}".html_safe
+        - if order_detail.problem?
```

Comment:
Do we need this conditional? I'm pretty sure all the orders that get rendered on this page are problems. I think we just had the check before to deal with deciding which message to display.